projects/text_image_video_to_text/README.md

```diff
@@ -0,0 +1,127 @@
+# Qwen2-VL Model Doc for Align-Anything
+
+## Environment setup
+
+Currently, the Transformer in pypi is not updated to support the latest Qwen2-VL model, so we need to install the forked stable version of the repo by running:
+pip install git+https://github.com/huggingface/transformers.git
```

Comment:
```bash
pip install git+https://github.com/huggingface/transformers.git
```

Reply:
Fixed in https://github.com/PKU-Alignment/align-anything/pull/68/commits/822f056c556cb35520f79ec5742e9b0b7301dd06